docs/common/getting_started/python.md

```diff
@@ -52,43 +52,16 @@ The Ephys and Imaging pipelines have subtly different requirements. Therefore, w
     * `conda install graphviz python-graphviz pydotplus ipykernel`
     * `conda install datajoint -c conda-forge`
 
-##### Working with Conda environments in Jupyter
+##### Working with Conda environments in Jupyter lab
 
-Jupyter is a popular interactive tool for working with Python. You can install Jupyter in your base Conda environment and use it with other environments that you have created on your computer given these steps:
+Jupyter lab is a popular interactive tool for working with Python. You can install Jupyter in your conda environment:
 
-* In the environment _you wish to make available_ (e.g. `ephys`):
-  * `conda activate ephys`
-  * `conda install ipykernel`
-* In the environment *from which you run Jupyter* (e.g. `base`):
-  * `conda deactivate`
-  * `conda install juypter nb_conda_kernels`
-* When you run Jupyter, you can select the Python Kernel matching the `conda env` name you wish to operate on
-
-##### Working with Conda Environments in Spyder
-
-`Spyder` is a popular Python development environment. It is natively installed with any *conda* Python distribution. If you do not have it installed, you can install it with either `conda` or `pip` (it is a Python package like any other)
-
-```python
-conda install spyder
-```
-
-
-
-Spyder does not directly support either `conda` environments, or the older styles `venv` virtual environments. However, you can work with them anyway in one of two ways:
-
-* Install `spyder` into the environment you wish to use, and use the resulting binary to run `spyder`
-* Install `spyder-kernels` into the environment you wish to use, and use `spyder` installed from the base environment.
-
-In the latter case, you must change `spyder` Preferences to use the appropriate Python interpreter. You can find the correct path by running the following code _inside the environment you wish to use_:
-
-```python
-python -c "import sys; print(sys.executable)"
-```
-
-And then copying this path to the provided textbox in `Preferences > Python Interpreter > Use the following interpreter`
-
-Numerous other IDEs support Python. Common examples include PyCharm and Visual Studio Code
+  * `conda activate ephys` (or `conda activate imaging`)
```

Comment:
Not necessary, Jupyter can (and, imo, should) be installed in the one environment only, preferably `base`, with other environments made available via `ipykernel` and `nb_conda_kernels`. It's pretty much the _only_ package that I strongly recommend be installed in the `base` environment only

Installing and running Jupyter from inside a single environment is both inefficient (i.e. multiple copies installed), inflexible (you need an independent Jupyter server for each environment you're using), and worst of all, actively encourages bad practice (the annoyance of switching from one server to another encourages users to stick to a single environment).

Reply:
Updated in commit 6b7a4c2